lib/screen/migration/key_sync_page.dart

```diff
@@ -8,32 +8,54 @@
 import 'package:autonomy_flutter/screen/app_router.dart';
 import 'package:autonomy_flutter/screen/migration/key_sync_bloc.dart';
 import 'package:autonomy_flutter/screen/migration/key_sync_state.dart';
-import 'package:autonomy_flutter/view/au_filled_button.dart';
+import 'package:autonomy_flutter/util/au_icons.dart';
+import 'package:autonomy_flutter/util/constants.dart';
+import 'package:autonomy_flutter/util/style.dart';
+import 'package:autonomy_flutter/util/ui_helper.dart';
 import 'package:autonomy_flutter/view/au_radio_button.dart';
 import 'package:autonomy_flutter/view/back_appbar.dart';
+import 'package:autonomy_flutter/view/primary_button.dart';
 import 'package:autonomy_flutter/view/responsive.dart';
 import 'package:autonomy_theme/autonomy_theme.dart';
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 
-class KeySyncPage extends StatelessWidget {
-  const KeySyncPage({Key? key}) : super(key: key);
+class KeySyncPage extends StatefulWidget {
+  const KeySyncPage({super.key});
+
+  @override
+  State<KeySyncPage> createState() => _KeySyncPageState();
+}
+
+class _KeySyncPageState extends State<KeySyncPage> {
+  late String _selectedKeychain;
```

Comment:
can we move this into `KeySyncState`. We can interact with state by define & listen new events in `KeySyncBloc`.

Reply:
We can get rid of all the `setState` in side this page.